dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/TenantController.java

```diff
@@ -90,12 +88,11 @@
     @ApiException(CREATE_TENANT_ERROR)
     public Result createTenant(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
                                @RequestParam(value = "tenantCode") String tenantCode,
-                               @RequestParam(value = "tenantName") String tenantName,
                                @RequestParam(value = "queueId") int queueId,
                                @RequestParam(value = "description", required = false) String description) throws Exception {
-        logger.info("login user {}, create tenant, tenantCode: {}, tenantName: {}, queueId: {}, desc: {}",
-                loginUser.getUserName(), tenantCode, tenantName, queueId, description);
-        Map<String, Object> result = tenantService.createTenant(loginUser, tenantCode, tenantName, queueId, description);
+        logger.info("login user {}, create tenant, tenantCode: {}, queueId: {}, desc: {}",
+                loginUser.getUserName(), tenantCode, queueId, description);
```

Comment:
logger.info("login user {}, create tenant, tenantCode: {}, queueId: {}, desc: {}",
                StringUtils.replaceNRTtoUnderline(loginUser.getUserName()),
                StringUtils.replaceNRTtoUnderline(tenantCode),
                queueId,
                StringUtils.replaceNRTtoUnderline(description));

Reply:
SonarCloud is still producing errors